src/Microsoft.Identity.Client/Cache/CacheFallbackOperations.cs

```diff
@@ -300,12 +297,12 @@ internal static class CacheFallbackOperations
             ICoreLogger logger,
             ILegacyCachePersistence legacyCachePersistence,
             string preferredEnvironment,
-            ISet<string> environmentAliases,
+            IEnumerable<string> environmentAliases,
             string clientId,
             string upn,
             string uniqueId)
         {
-            var adalRts = GetAllAdalEntriesForMsal(logger, legacyCachePersistence, environmentAliases, clientId, upn, uniqueId);
+            List<MsalRefreshTokenCacheItem> adalRts = GetAllAdalEntriesForMsal(logger, legacyCachePersistence, environmentAliases, clientId, upn, uniqueId);
```

Comment:
> List<MsalRefreshTokenCacheItem>  [](start = 11, length = 33)

is type specification preferred now?

Reply:
VS suggests that if the variable type is obvious, we should use 'var'. If it's not obvious, we should use the type. Probably not a hard rule, I just needed to see the type of adalRts so it's clear that the type isn't obvious :)